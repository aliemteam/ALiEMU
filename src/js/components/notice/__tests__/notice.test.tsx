import React from 'react';
import renderer from 'react-test-renderer';

import Notice from '../index';

it('renders correctly', () => {
    const tree = renderer.create(<Notice>Hello World</Notice>);
    expect(tree).toMatchSnapshot();
});
