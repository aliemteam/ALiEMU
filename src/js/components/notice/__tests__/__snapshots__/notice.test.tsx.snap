// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className=""
>
  <div>
    Hello World
  </div>
</div>
`;
